src/main/java/graphql/schema/impl/SchemaUtil.java

```diff
@@ -1,11 +1,24 @@
-package graphql.schema;
+package graphql.schema.impl;
```

Comment:
Again again I moved this to a schem.impl package just to clean things up. Its kinda of a private thing and not general purpose